build-system/tasks/runtime-test.js

```diff
@@ -719,6 +723,7 @@ gulp.task('test', 'Runs tests', preTestTasks, function() {
     'a4a': '  Runs all A4A tests',
     'coverage': '  Run tests in code coverage mode',
     'headless': '  Run tests in a headless Chrome window',
+    'chrome_canary': 'Running tests on Chrome Canary.',
```

Comment:
nit: Move this next to the other browsers too.

wording nit: This gets printed for `gulp help`, so reword it to `Runs tests on Chrome Canary` to go with all the other help messages.

Reply:
Moved and phrasing change made.